### DISCUSSION TITLE
Setup UnoCSS for Vue.

### DISCUSSION THREAD

**Discussion by louiss0** [nrwl/nx#19888] [Feature Requests]
Can we please get a setup UnoCSS plugin for Nx? UnoCSS is a Utility class generator just like Tailwind. It's used by a lot of Vue devs.
[here](https://unocss.dev)

**Comment by ppusapati** [comment]
Any update on this?

**Reply by louiss0** [reply]
> Any update on this?

Nope they don't seem to think it's worth it

**Comment by louiss0** [comment]
Nope

**Comment by ppusapati** [comment]
I have a workaround. 

have created uno.config.ts file in root folder and added it in tsconfig.base.json

```
  "include": ["./unocss.config.ts"],
```

and in uno.config.ts have added filesystem in content as 

```
 filesystem:[
      '{projectRoot}/**/*.[jt]s?(x)?(.snap)',
    ],

```

this loads all classes declared in all applications in apps folder. but, am not sure if it works with HMR and  presetTagify
